tests/config_change/config_change_suite_test.go

```diff
@@ -94,7 +94,7 @@ var _ = Describe(testName, func() {
 			ns.WaitForOutputAndLog(step, k, "Ready", 1800)
 
 			step = "change the config"
-			json = "{\"spec\": {\"config\": {\"cassandra-yaml\": {\"file_cache_size_in_mb\": 123}}}}"
+			json = "{\"spec\": {\"config\": {\"cassandra-yaml\": {\"file_cache_size_in_mb\": 123}, \"jvm-options\": {\"garbage_collector\": \"CMS\"}}}}"
```

Comment:
should be jvm-server-options for DSE though?

Reply:
Yes, it should be `jvm-server-options`. I am in a bit of a catch 22 situation. I cannot pull the DSE images (see #25). I can only run the test against Cassandra 3.11.x. I will change that back to `jvm-server-options` prior to if/when the PR is otherwise ready to merge.